apps/smpltmr/clkinfo.js

```diff
@@ -71,13 +71,19 @@
     ]
   };
 
+  const restoreMainItem = function(clkinfo) {
+    clkinfo.menuB = 0;
+    // clock info redraws after this
```

Comment:
I just saw you have:

```
 const blur = () => {
    options.focus=false;
    delete Bangle.CLKINFO_FOCUS;
    options.redraw();
    const itm = menu[options.menuA].items[options.menuB];
    if (itm.blur) itm.blur(options);
  };
```

So does it actually redraw?

Reply:
Yes - I ran out of time to push this morning, [pushed now](https://github.com/espruino/BangleApps/pull/3234/commits/736b9f0433ad76d2867bd2d37401c1cc382d5b42)